index.html

```diff
@@ -244,14 +244,79 @@ <h2>
               </div>
             </li>
           </ul>
-          <ul class="info">
-            <a href="#">Frameworks</a>
-          </ul>
-          <ul class="info">
-            <a href="#">Skills</a>
-          </ul>
+          
         </section>
       </section>
     </section>
+    
+    </div>
+    <section class="bg_color">
+      
+      
+      <div class="contact_section">
+        
+       
+        <div class="contact_form">
+          
+          <header>
+            <h3>Contact me</h3>
+          </header>
+          <div class="text3">
+            
+            <h2>
+              If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it.
+            </h2>
+          </div>
+        <form     
+          action="https://formspree.io/f/xpznodvr"
+          method="post"
+        >
+        
+          <ul>
+            
+            <li>
+              <input
+              class="text"
+                type="text"
+                name="fname"
```

Comment:
Great job implementing some of the changes required by the previous reviewer, As mentioned, kindly add the background images to your project to match the [Figma ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=tiRt5v6tUpLB21lK-0) design.

| Your Design | Figma Design |
| :---:   | :---: | 
|<img width="384" alt="image" src="https://user-images.githubusercontent.com/73854999/206851654-ca14b3f7-0730-4c1e-9d4b-f657b30e85ac.png">|<img width="384" alt="image" src="https://user-images.githubusercontent.com/73854999/206851680-0ea270f4-4672-4cce-95e2-c7348e6115c4.png">|